apps/docs/content/guides/platform/manage-your-usage/disk-size.mdx

```diff
@@ -0,0 +1,110 @@
+|                               |           |            |
+| Compute Hours Micro Project 1 | 744 hours | $10        |
+| Disk Size Project 1           | 8 GB      | $0         |
+|                               |           |            |
+| Compute Hours Micro Project 2 | 744 hours | $10        |
+| Disk Size Project 2           | 8 GB      | $0         |
+|                               |           |            |
+| Compute Hours Micro Project 3 | 744 hours | $10        |
+| Disk Size Project 3           | 50 GB     | $5.25      |
+|                               |           |            |
+| **Subtotal**                  |           | **$50.25** |
+| Compute Credits               |           | -$10       |
+| **Total**                     |           | **$40.25** |
+
+### io2
+This disk type is billed from the first byte of provisioned disk, meaning for 66 GB across all projects.
+
+| Line Item                     | Units     | Costs      |
+|-------------------------------|-----------|------------|
+| Pro Plan                      | 1         | $25        |
+|                               |           |            |
+| Compute Hours Micro Project 1 | 744 hours | $10        |
+| Disk Size Project 1           | 8 GB      | $1.56      |
+|                               |           |            |
+| Compute Hours Micro Project 2 | 744 hours | $10        |
+| Disk Size Project 2           | 8 GB      | $1.56      |
+|                               |           |            |
+| Compute Hours Micro Project 3 | 744 hours | $10        |
+| Disk Size Project 3           | 50 GB     | $9.75      |
+|                               |           |            |
+| **Subtotal**                  |           | **$67.87** |
+| Compute Credits               |           | -$10       |
+| **Total**                     |           | **$57.87** |
+
+## View Disk size usage
+You can view Disk size usage on the [organization's usage page](https://supabase.com/dashboard/org/_/usage). The page shows the usage of all projects by default. To view the usage for a specific project, select it from the dropdown.
+<Image
+  alt="Usage page navigation bar"
+  src={{
+    light: '/docs/img/guides/platform/usage-navbar--light.png',
+    dark: '/docs/img/guides/platform/usage-navbar--dark.png',
+  }}
+  zoomable
+/>
+
+In the Disk size section, you can see how much disk size your projects have provisioned.
+<Image
+  alt="Usage page Disk Size section"
+  src={{
+    light: '/docs/img/guides/platform/usage-disk-size--light.png',
+    dark: '/docs/img/guides/platform/usage-disk-size--dark.png',
+  }}
+  zoomable
+/>
+
+### Disk size distribution
+To see how your disk usage is distributed across Database, WAL, and System categories, refer to [Disk size distribution](/docs/guides/platform/database-size#disk-size-distribution).
+
+## Reduce Disk size
+To see how you can downsize your disk, refer to [Reducing disk size](/docs/guides/platform/database-size#reducing-disk-size)
```

Comment:
Not sure what's better here. Links to the sections or showing the actual content and eventually use partials. @charislam any thoughts on how you'd prefer that throughout the docs (in general)?

Reply:
I think this works well as a link. In general, it's quite context-specific: if the content is very related to the actual page, probably better as a partial (especially if you can't imagine ever reading the page without _also_ wanting to know about that other content). If it's not super related, a link is better. In this case, I'd categorize as "not super related" since the rest of the page is more of a reference defining disk size and its billing, and the linked content is a list of tips for reducing disk size.